### PR TITLE
Adding authentication info to the migrations API

### PR DESCRIPTION
**Pull request by bitoiu**

### CODE REVIEW

File: content/v3/migration/migrations.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ title: Migrations
 
 {:toc}
 
+The Migrations API *is only available to [authenticated](https://developer.github.com/v3/#authentication) organization owners*. Non-organization owners will receive a `404` response if they try to access it.
```
Comment (jonmagic) [inline content/v3/migration/migrations.md]: IO'm not sure `organization owner` is correct anymore? I think `organization admin` may be better but I'm not sure.

**Comment by jonmagic** [comment]
> Although I'm not 100% sure that you need to be an owner, it did work when I tested it in these conditions.

You do need to be an organization admin: https://github.com/github/github/blob/4fd81e5955463fbb996c62990ae78847b26e617f/app/api/access_control/migration_access_dependency.rb#L7-L9

:+1: with the small note about doing owner vs admin.

**Comment by bitoiu** [comment]
Hey @jonmagic, thanks for jumping on this so quickly :bow:

> You do need to be an organization admin

After direct orgs, the only way (I see) to become an organization admin is to set `admin` as the default membership access:

screen shot 2016-03-22 at 18 03 34

Other than this configuration, is there any other way to be an Org admin? I ask this because I'm afraid users will think that repository administration will be enough. Looking at the code it seems like even if I have admin access to `repository A` I still need to be an org admin to use the API.

---

#### Two suggestions:

We explicitly mention ownership and organization admin access:

> The Migrations API *is only available to [authenticated](https://developer.github.com/v3/#authentication) [organization owners](https://help.github.com/articles/permission-levels-for-an-organization/) or admins*. Other users will receive a `404` response if they try to access it.

or we just mention organization admin access, assuming the user gets that ownership includes it:

> The Migrations API *is only available to [authenticated](https://developer.github.com/v3/#authentication) organization admins*. Non-organization admins will receive a `404` response if they try to access it.

A few notes:
 - used `admin(s)` everywhere since that is our official wording for the role
 - didn't find a good article to link to "organization admin". I don't want to link the [repository access levels](https://help.github.com/articles/repository-permission-levels-for-an-organization/) because it might imply that admin at the repository level would work.

Do you like any of the options @jonmagic?

:octocat::heart:

**Comment by jonmagic** [comment]
I think I was wrong about "Owner", I was getting confused with old owners team stuff... We still have owners, they just aren't teams anymore, so I think you had it right the first time :+1: Sorry for the confusion and thank you for the detailed explanation above :heart:

**Comment by jonmagic** [comment]
2016-03-22 at 11 43 am

**Pull request closed without merging**